src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarDatePicker.Properties.cs

```diff
@@ -63,7 +63,7 @@ public string DayOfWeekFormat
 #endif
 			object Description
 		{
-			get => (string)GetValue(DescriptionProperty);
+			get => GetValue(DescriptionProperty);
```

Comment:
This was a bug right?

Reply:
Yes, that was copy paste bug I think :-)